data-raw/watershed/floodplain_utils.R

```diff
@@ -0,0 +1,237 @@
+library(tidyverse)
```

Comment:
why do you have two versions of this script? `watershed/floodplain_utils.R` and `watershed/floodplain/floodplain_utils.R` I wouldn't nest it a floodplain directory

Reply:
Un-nested